Mu2eG4/fcl/surfaceCheck.fcl

```diff
@@ -36,5 +36,6 @@ services.SeedService.maxUniqueEngines :  20
 physics.producers.generate.inputfile : "Offline/EventGenerator/defaultConfigs/surfaceCheck.txt"
 
 services.GeometryService.inputFile : "Offline/Mu2eG4/geom/geom_SurfaceCheck.txt"
+services.GeometryService.bFieldFile : "Offline/Mu2eG4/geom/bfgeom_reco_v01.txt"
```

Comment:
Doesn't thin effective add a field to this job when there wasn't one before?  Why not no_field?